packages/js/src/Modules/Verto/BaseSession.ts

```diff
@@ -188,11 +188,6 @@ export default abstract class BaseSession {
   /**
    * Attaches an event handler for a specific type of event.
    *
-   * @param eventName Event name.
-   * @param callback Function to call when the event comes.
-   *
-   * @return The client object itself.
-   *
    * ## Examples
```

Comment:
You could change this to be `@examples` so that you don't have to provide than on top of `@param`. You will still need to have `@param` after `## Events` though.